client/layout/index.jsx

```diff
@@ -81,6 +82,17 @@ const Layout = createReactClass( {
 			return <MasterbarLoggedOut sectionName={ this.props.section.name } />;
 		}
 
+		// miminmal checkout, less distraction
+		if ( this.props.section.name === 'checkout' ) {
+			return (
+				<MasterbarCheckout
+					user={ this.props.user }
+					section={ this.props.section.group }
+					sites={ this.props.sites }
```

Comment:
The `sites` prop is not used by the `MasterbarCheckout` component and can be omitted.

So can `user`, after the `user.get().site_count` usage is replaced by its Redux equivalent.